### PR TITLE
Add curation keyboard shortcuts

### LINKED ISSUE #364: [Feature] Add keyboard shortcuts to the curation interface

### PR DESCRIPTION
**Pull request by adamltyson**
Requires #365 to be merged first

Closes #364

### CODE REVIEW

File: cellfinder/napari/curation.py [hunk 1]
```diff
@@ -373,16 +384,45 @@ def save_training_data(
...
             self.update_status_label("Ready")
 
+    def __prep_directories_for_save(self):
+        self.yaml_filename = self.output_directory / "training.yml"
+        self.cell_cube_dir = self.output_directory / "cells"
+        self.no_cell_cube_dir = self.output_directory / "non_cells"
+
+        self.__delete_existing_saved_training_data()
+
+    def __delete_existing_saved_training_data(self):
```
Comment (alessandrofelder) [inline cellfinder/napari/curation.py:415]: For my understanding, why do functions in the file start with two underscores??

**Review by alessandrofelder — APPROVED** [review]
Looks good!

Just making a note here for our future selves and other interested people, that this PR mainly
* adds keyboard shortcuts
but also
* replaces local functions with `brainglobe-utils` equivalents
* removes tests for those functions
* makes tiny improvements to explicitness of code
* adds a "overwrite existing data?" warning.

File: cellfinder/napari/curation.py (hunk 1 above)
Comment (adamltyson) [reply cellfinder/napari/curation.py:415]: Double underscores [are as close to private as you can get in Python](https://docs.python.org/3/tutorial/classes.html#private-variables). Not sure why they're used here and not anywhere else in BrainGlobe though.

Comment (alessandrofelder) [reply cellfinder/napari/curation.py:415]: Ah cool! Didn't know. From reading that it sounds like the use case is mainly to make it impossible to override this in subclasses. We currently don't have any subclasses of `CurationWidget`, and for consistency it would be good to not use it here, but then again not sure it's enough of a priority right now :thinking: 

TLDR I suggest leaving for now?

Comment (adamltyson) [reply cellfinder/napari/curation.py:415]: Fine by me.

**Comment by adamltyson** [comment]
The tests are failing now, but is this due to #368?

**Comment by adamltyson** [comment]
Yeah there's no rush for this
